server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -50,6 +52,7 @@
 import io.crate.testing.UseJdbc;
 
 @IntegTestCase.ClusterScope(numDataNodes = 2)
+@UseRandomizedOptimizerRules(value = 1, disablePercentage = 1, alwaysKeep = RewriteInsertFromSubQueryToInsertFromValues.class)
```

Comment:
I think `RewriteInsertFromSubQueryToInsertFromValues.class` is a bit special because we've different subquery/from-values semantics. It's not really an optimization rule in the sense of merely improving performance. Maybe we should actually make that more concrete and not use the rule framework for it or at least disallow disabling that rule.